play/symphony-conductor/index.html

```diff
@@ -200,16 +201,16 @@ <h3 id="label10">Video Tutorial</h3> <h3 id="label11">Examples</h3>
 
                 <button onclick="myFunction(`myInput2`)" id="exampleButton2">Copy (Complex Level)</button>
 
-                <iframe width="449" height="310" src="https://www.youtube.com/embed/0kg-ZQXIpFY" id="tutorialVideo" >
-            </iframe>
+                <iframe width="449" height="310" src="https://www.youtube.com/embed/fRTmOHbHCJM" id="tutorialVideo">
+                </iframe>
 
                 <script>
                     function myFunction(name) {
```

Comment:
This is not a descriptive function name and this could also be in app.js

Reply:
I improved the variable names and put the function into app.js